internal/kubernetes/helm.go

```diff
@@ -234,13 +234,13 @@ func patchHelmDirectories(client *HelmClient) error {
 		}
 	}
 
-	pathCache := []string{usr, ".cache", "helm", "repository"}
-	cachePath, okCache := store.PresentDirectory(pathCache)
-	if !okCache {
-		if _err := store.CreateDirectory(pathCache); _err != nil {
-			return _err
-		}
-	}
+	// pathCache := []string{usr, ".cache", "helm", "repository"}
+	// cachePath, okCache := store.PresentDirectory(pathCache)
+	// if !okCache {
+	// 	if _err := store.CreateDirectory(pathCache); _err != nil {
+	// 		return _err
+	// 	}
+	// }
```

Comment:
#### Solution
In Macos Systems this cache folder is not able to work properly
and as it is not that important we can skip setting this setting